src/main/java/com/minecolonies/colony/Colony.java

```diff
@@ -467,17 +411,16 @@ public void onWorldTick(TickEvent.WorldTickEvent event)
         }
 
         //  Tick Buildings
-        for (Building building : buildings.values())
+        for (AbstractBuilding building : buildings.values())
         {
             building.onWorldTick(event);
         }
 
         workManager.onWorldTick(event);
     }
 
-
     /**
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) First sentence should end with a period. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=checkstyle%3AJavadoc_Style)